packages/cli/src/init.ts

```diff
@@ -34,6 +34,12 @@ export async function promptNewIntegration(dirPath: string): Promise<void> {
             message: 'Title of the integration:',
             initial: (prev) => prev || '',
         },
+        {
+            type: 'text',
+            name: 'organization',
+            message: 'ID of the organization this integration is owned by',
```

Comment:
It can also be the GitBook `subdomain`, probably worth mentioning it

Reply:
Nice, didn't know you already updated that. I'll update and merge